src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -227,18 +228,25 @@ public static EnergyBalanceInfo ComputeEnergyBalance(IReadOnlyList<OrganelleTemp
     ///   If true, then the required input compounds to run at the given energy balance are stored per energy producer
     /// </param>
     /// <param name="cache">Auto-Evo Cache for speeding up the function</param>
+    /// <param name="result">
+    ///   Energy balance info to add the new info to. To be used for calculating multiple cells' balances
+    /// </param>
     public static EnergyBalanceInfo ComputeEnergyBalance(IReadOnlyList<OrganelleTemplate> organelles,
```

Comment:
I think the return value should be removed and this refactored so that the balance info is mandatory value to pass in. That way the usage would be clearer.